package.json

```diff
@@ -27,13 +27,13 @@
   "peerDependencies": {
     "bedrock": "2.0.0 - 4.x",
     "bedrock-jsonld-document-loader": "^1.0.0",
-    "bedrock-security-context": "^3.0.0"
+    "bedrock-security-context": "^4.1.0"
   },
   "engines": {
-    "node": ">=6.3.0"
+    "node": ">=14.0.0"
```

Comment:
Why this change and not node 12?

Reply:
https://github.com/digitalbazaar/bedrock-ledger-context/pull/14/commits/3b1d5b5c4aee9781f537352b936e27a7c1abcf16